homeworks/andrew.markovskiy_MarkoAndrew93/0-intro/GamesProject/cards.html

```diff
@@ -0,0 +1,265 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="../GamesProject/css/styles.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500;700&family=Roboto:wght@400;700&family=Young+Serif&display=swap" rel="stylesheet">
+        <title>GamesProject</title>
+    </head>
+    <body>
+        <header>
+            <section class="header-conteiner">
+                <div class="header-line">    
+                    <a class="headder-logo_img" href="">
+                        <img class="header-logo" src="../GamesProject/img/logo.png" alt="">
+                    </a>
+                    <nav class="header-menu">
+                        <ul class="header-menu_list">
+                            <li class="header-menu_item">
+                                <a class="header-menu_link" href="">Games</a>
+                            </li>
+                            <li class="header-menu_item">
+                                <a class="header-menu_link" href="../GamesProject/index.html">About</a>
+                            </li>
+                            <li class="header-menu_item">
+                                <a class="header-menu_link" href="">Login</a>
+                            </li>
+                        </ul>
+                    </nav>
+                </div>
+            </section>
+        </header>
+        <main>
+            <h1 class="cards-title">Study project "Games"</h1>
+            <div class="conteiner-cards">
+                <div class="conteiner-cards_item">
```

Comment:
you have multiple similar cards; it's essentially a list of cards. Don't use `div` if you can use semantic tag

For ex. :

```
<ul class="games__list">      //   (games__body - in your code)
   <li class="games__card">
       All card here
       ....
   </li>
</ul>
```